### PR TITLE
Blazor Pre6 Sweep 1

### PR DESCRIPTION
**Pull request by guardrex**
Addresses #10717 

I'm not going to put all of the updates on just one PR. I'll set these up with about five topics per PR so that they're a bit more manageable to review.

These are likely going to be reviewed on the diff, but let me know if you want the internal review links.

### CODE REVIEW

**Comment by guardrex** [comment]
@blowdart There's just one line here that I think you should review. We've been saying the following ...

> .NET code executed via WebAssembly in the browser runs in the same trusted sandbox as JavaScript, which virtually eliminates the opportunity for an app to perform malicious actions on the client machine.

However, I spoke with a security MVP at the conference who told me that the "sandbox" where the code runs is regularly patched for security issues that arise. Based on his opinion, it seems that "virtually eliminates" is a bit strong. I change that just a hair to ...

> ... same trusted sandbox as JavaScript, which reduces the opportunity for an app to perform malicious actions on the client machine.

Does moving a bit away from "virtually eliminates," as I do on the PR here, make sense? If not, do you have alternate language for this line, or should I leave it alone as "virtually eliminates"?

File: aspnetcore/blazor/hosting-models.md [hunk 1]
```diff
@@ -25,34 +25,34 @@ The principal hosting model for Blazor is running client-side in the browser on
...
+The client-side hosting model offers several benefits:
 
-* Has no .NET server-side dependency.
-* Fully leverages client resources and capabilities.
-* Offloads work from the server to the client.
-* Supports offline scenarios.
+* There's no .NET server-side dependency. The app is fully functioning after downloaded to the client.
+* Client resources and capabilities are fully leveraged.
+* Work is offloaded from the server to the client.
+* An ASP.NET Core server isn't required to serve the app. Serverless deployment scenarios are possible (for example, serving the app from a CDN).
```
Comment (scottaddie) [inline aspnetcore/blazor/hosting-models.md]:
```suggestion
* An ASP.NET Core web server isn't required to host the app. Serverless deployment scenarios are possible (for example, serving the app from a CDN).
```

File: aspnetcore/blazor/index.md [hunk 2]
```diff
@@ -17,25 +17,30 @@ By [Daniel Roth](https://github.com/danroth27) and [Luke Latham](https://github.
...
+* Define flexible UI rendering logic.
+* Handle user events.
+* Can be nested and reused.
+* Can be shared and distributed as [Razor class libraries](xref:razor-pages/ui-class) or [NuGet packages](/nuget/what-is-nuget).
+
+The component class is usually written in the form of a [Razor](xref:mvc/views/razor) markup page with a *.razor* file extension. Components in Blazor are sometimes referred to as *Razor components*. Razor is a syntax for combining HTML markup with C# code designed for developer productivity. Razor allows you to switch between HTML markup and C# in the same file with [IntelliSense](/visualstudio/ide/using-intellisense) support. Razor Pages and MVC also use Razor. Unlike Razor Pages and MVC, which are built around a request/response model, components are used specifically for client-side UI logic and composition.
```
Comment (scottaddie) [inline aspnetcore/blazor/index.md]: Should the *Razor components* reference be replaced with Blazor server-side?

File: aspnetcore/tutorials/build-your-first-blazor-app.md [hunk 3]
```diff
@@ -14,32 +14,32 @@ By [Daniel Roth](https://github.com/danroth27) and [Luke Latham](https://github.
 
 This tutorial shows you how to build and modify a Blazor app.
 
-Follow the guidance in the <xref:blazor/get-started> article to create a Blazor project for this tutorial.
+Follow the guidance in the <xref:blazor/get-started> article to create a Blazor project for this tutorial. Name the project **ToDoList**.
```
Comment (scottaddie) [inline aspnetcore/tutorials/build-your-first-blazor-app.md]:
```suggestion
Follow the guidance in the <xref:blazor/get-started> article to create a Blazor project for this tutorial. Name the project *ToDoList*.
```

File: aspnetcore/blazor/index.md (hunk 2 above)
Comment (guardrex) [reply aspnetcore/blazor/index.md]: Not here ... the point is to say that sometimes components are called "Razor components."

Comment (guardrex) [reply aspnetcore/blazor/index.md]: Alternatively to this, we could say ...

> Components in Blazor are formally referred to as *Razor components*.

I kind'a like that better actually.

Comment (guardrex) [reply aspnetcore/blazor/index.md]: @danroth27 I don't think u need to review this PR. The vast majority of updates are nits we've already discussed. On this point tho :point_up:, how would u like to clarify the text?

Comment (danroth27) [reply aspnetcore/blazor/index.md]: > Components in Blazor are formally referred to as Razor components.

I like this better too.

**Comment by guardrex** [comment]
Look good, @scottaddie?

@blowdart Let me know, and I'll react on another PR. I'll be putting in at least one more *Blazor Sweep* PR today, and I'll react on that security lingo there if needed.

**Comment by blowdart** [comment]
Hmm how about

".NET code executed via WebAssembly in the browser runs in the browser's JavaScript sandbox, with all the protections the sandbox provides again malicious actions on the client machine."

File: aspnetcore/blazor/components.md [hunk 4]
```diff
@@ -60,7 +60,7 @@ Components are ordinary C# classes and can be placed anywhere within a project.
 Use components with existing Razor Pages and MVC apps. There's no need to rewrite existing pages or views to use Razor components. When the page or view is rendered, components are prerendered&dagger; at the same time. 
 
 > [!NOTE]
-> &dagger;Server-side prerendering is enabled for Blazor server-side apps by default. Client-side Blazor apps will support prerendering in the upcoming Preview 5 release. For more information, see [Update templates/middleware to use MapFallbackToPage/File](https://github.com/aspnet/AspNetCore/issues/8852).
+> &dagger;Server-side prerendering is enabled for Blazor server-side apps by default. Blazor client-side apps will support prerendering in the upcoming Preview 5 release. For more information, see [Update templates/middleware to use MapFallbackToPage/File](https://github.com/aspnet/AspNetCore/issues/8852).
```
Comment (scottaddie) [inline aspnetcore/blazor/components.md]: Remove the reference to "Preview 5"

Comment (guardrex) [reply aspnetcore/blazor/components.md]: Ah ... wait ... I'll get that on the next one. The Components topic will be on the next PR coming up by EOD.

**Pull request merged**